app-android/src/main/java/com/ustadmobile/port/android/view/ContentEntryDetailOverviewFragment.kt

```diff
@@ -346,4 +376,438 @@ class ContentEntryDetailOverviewFragment: UstadDetailFragment<ContentEntryWithMo
+            item {
+                LocallyAvailableRow()
+            }
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        item {
+            Divider(thickness = 1.dp)
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        item {
+            QuickActionBarsRow(
+                uiState = uiState,
+                onClickMarkComplete = onClickMarkComplete,
+                onClickDelete = onClickDelete,
+                onClickManageDownload = onClickManageDownload
+            )
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        item {
+            Text(text = "Locally Available")
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        item {
+            Divider(thickness = 1.dp)
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        item {
+            Text(text = uiState.contentEntry?.description ?: "")
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        item {
+            Divider(thickness = 1.dp)
+        }
+
+        item {
+            Spacer(modifier = Modifier.height(10.dp))
+        }
+
+        if (uiState.translationVisibile){
+            item {
+                Text(text = stringResource(id = R.string.also_available_in))
+            }
+
+            items(
+                uiState.availableTranslations.size
+            ){
+                ListItem(
+                    modifier = Modifier.clickable {
+                        onClickTranslation()
+                    },
+                    text = { Text(uiState.availableTranslations[it].language?.name ?: "") },
+                )
+            }
+        }
+    }
+}
+
+@Composable
+fun ContentDetails(
+    uiState: ContentEntryDetailOverviewUiState = ContentEntryDetailOverviewUiState(),
+){
+    Row {
+
+        Box(
+            modifier = Modifier.weight(0.3F)
+        ) {
+            LeftColumn(
+                uiState = uiState
+            )
+        }
+
+        Spacer(modifier = Modifier.width(10.dp))
+
+        Box(
+            modifier = Modifier.weight(0.7F)
+        ) {
+            RightColumn(
+                uiState = uiState
+            )
+        }
+    }
+}
+
+@Composable
+fun LeftColumn(
+    uiState: ContentEntryDetailOverviewUiState = ContentEntryDetailOverviewUiState(),
+){
+    Column{
+        Image(painter = painterResource(id = R.drawable.book_24px),
+            contentDescription = "",
+            modifier = Modifier.size(110.dp),
+            contentScale = ContentScale.Crop
+        )
+
+        Row(
+            verticalAlignment = Alignment.CenterVertically
+        ) {
+            if (uiState.scoreProgressVisible){
+                LinearProgressIndicator(
+                    progress = 0.6F,
```

Comment:
What to display here must be based on the scoreProgress, should not use hard-coded demo constants here! The progress bar should only show when there is progress (e.g. > 0). Also the checkbox should only show if the user has passed/completed. The cross should be shown if the user failed. If the user has not passed, completed, or failed, then there is no icon at the end of the checkbox.